packages/slate-hyperscript/test/default/perserve-keys-in-right-text.js

```diff
@@ -0,0 +1,72 @@
+/** @jsx h */
+
+import assert from 'assert'
+
+import h from '../..'
+
+export const input = (
+  <document>
+    <block type="paragraph">
+      Cat <inline type="link">is</inline>
+      <text key="a"> cute</text>
+    </block>
+  </document>
+)
+export function assertation() {
```

Comment:
I think `function test()` is a better name than `function assertation()`.

Reply:
Great suggestion.  I make the rename in 98884c7.